packages/components/src/tab-panel/index.tsx

```diff
@@ -101,7 +101,7 @@ export function TabPanel( {
 	// to show the `tab-panel` associated with the clicked tab.
 	const activateTabAutomatically = (
 		_childIndex: number,
-		child: HTMLButtonElement
+		child: HTMLElement
```

Comment:
To prevent a type error:

```sh
packages/components/src/tab-panel/index.tsx:158:5 - error TS2322: Type '((_childIndex: number, child: HTMLButtonElement) => void) | undefined' is not assignable to type '((index: number, focusable: HTMLElement) => void) | undefined'.
  Type '(_childIndex: number, child: HTMLButtonElement) => void' is not assignable to type '(index: number, focusable: HTMLElement) => void'.
    Types of parameters 'child' and 'focusable' are incompatible.
      Type 'HTMLElement' is missing the following properties from type 'HTMLButtonElement': disabled, form, formAction, formEnctype, and 13 more.

158     onNavigate={
        ~~~~~~~~~~

  packages/components/src/navigable-container/types.ts:29:2
    29  onNavigate?: ( index: number, focusable: HTMLElement ) => void;
        ~~~~~~~~~~
    The expected type comes from property 'onNavigate' which is declared here on type 'IntrinsicAttributes & BaseProps & { orientation?: "horizontal" | "vertical" | "both" | undefined; } & Omit<Pick<DetailedHTMLProps<HTMLAttributes<HTMLDivElement>, HTMLDivElement>, "key" | keyof HTMLAttributes<...>>, "as" | ... 1 more ... | keyof BaseProps> & RefAttributes<...>'
```